pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/RocksdbMetadataStore.java

```diff
@@ -220,6 +223,8 @@ private RocksdbMetadataStore(String metadataURL, MetadataStoreConfig metadataSto
         Path dataPath = FileSystems.getDefault().getPath(dataDir);
         try {
             Files.createDirectories(dataPath);
+            Set<PosixFilePermission> perms = PosixFilePermissions.fromString("rwx------");
+            Files.setPosixFilePermissions(dataPath, perms);
         } catch (IOException e) {
             e.printStackTrace();
```

Comment:
Could you help to change `log.error` ?

Reply:
> Could you help to change to `log.error` ?

We should throw exception here. I will fix this.